### ISSUE TITLE
Export MathML in ODT as LaTeX

### ISSUE THREAD

**Issue by memeplex** [jgm/pandoc#5602]
I have created the attached and zipped odt by exporting it from google docs. The text is just "Hi x^2" where x^2 is an squared x as an equation. Converting it to markdown using pandoc gives:

```
pandoc -t markdown Prueba.odt 
Hi ![](./ObjectReplacements/Object 2){width="0.1417in"
```

Inside the odt `Object 2/content.xml` is:

```
cat Object\ 2/content.xml 
<?xml version="1.0" encoding="UTF-8"?>
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><msup><mi>x</mi><mn>2</mn></msup><annotation encoding="StarMath 5.0">{x} ^ {2}</annotation></semantics></math>
```

that is, the equation was exported as MathML, not as an image. Now, I would have expected this to be exported as LaTeX in the markdown output.

Am I missing something?

Pandoc version: 2.2.1

[Prueba.zip](https://github.com/jgm/pandoc/files/3308587/Prueba.zip)

**Comment by memeplex** [comment]
BTW, this is the output when exporting as docx:

```
pandoc -t markdown Prueba.docx 
Hi $x^{2}$.
```

Much better!

**Comment by jgm** [comment]
I don't think the Odt reader currently parses math elements.
This would be a very useful addition.

Note, however, that we'd only be able to parse presentation mathml, not semantic mathml (you use the 'semantics' tag here).

**Comment by memeplex** [comment]
I barely understand mathml, let alone what semantic/presentation tags are. I can say that it works fine with docx though, so if you're doing the "presentation" stuff there I assume it should work here too.

**Comment by memeplex** [comment]
Maybe I should have mentioned that the equations are represented as mathml in docx too. I have verified this. I don't know about the semantic/presentation thing though.

**Comment by mb21** [comment]
Might work with #5606...

**Comment by jgm** [comment]
@memeplex No, docx doesn't use mathml.  It uses a different, XML-based math format.  texmath and pandoc can convert between these.

**Comment by memeplex** [comment]
Thanks!

**Comment by jgm** [comment]
Sorry, it was claimed in the PR that it fixed this issue, but it doesn't.
